src/main/kotlin/com/jetbrains/snakecharm/codeInsight/wrapper/WrapperStorage.kt

```diff
@@ -0,0 +1,59 @@
+package com.jetbrains.snakecharm.codeInsight.wrapper
+
+import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.components.PersistentStateComponent
+import com.intellij.openapi.components.State
+import com.intellij.openapi.components.Storage
+import com.intellij.util.xmlb.XmlSerializerUtil
+
+@State(name = "WrapperStorage", storages = [Storage("wrapper.xml")])
+class WrapperStorage : PersistentStateComponent<WrapperStorage> {
+
+    var wrapperStringStorage: MutableList<String> = mutableListOf()
```

Comment:
Public mutable list is exposed for now reason. `WrapperSteps` just replace the whole list. So the list should be immutable, i.e. `List<String>` at least.

Reply:
I think so too, but I was unable to get it to serialize when adding a `private` visibility modifier or making this a `val` instead of a `var`. The file `wrapper.xml` in which the data is stored just wouldn't get created. I figured IDEA needs it to be public and modifiable in order to be able to serialize it, but if you are certain that it is not so (I was unable to find any references in docs, as for examples from source code, IIRC it was mostly settings where fields were all public and modifiable, although of course I might have just not looked thoroughly enough), I could try to work around this issue.

Reply:
> when adding a private visibility 

I'm talking not about `val` or `var`, but about `MutableList` interface, which allows elements addition/deletion from it. If it isn't supposed that user could modify a list, better to keep in immutable, e.g. use `List` instead of `MutableList` in Kotlin.